### PR TITLE
Task8post comment

### PR DESCRIPTION
**Pull request by FameTii**
able to post comments, 
user hardcoded in

### CODE REVIEW

File: src/Api.js [hunk 1]
```diff
@@ -31,4 +31,14 @@ const getComments = (reviewId) => {
...
 
-export { getReviews, getReviewById, getComments };
+const postComment = (reviewId, comment, username) => {
+  return axios
+    .post(
+      `https://fame-boardgame-review-website.onrender.com/api/reviews/${reviewId}/comments`,
+      { username: username, body: comment }
+    )
+    .then(({ data }) => {
+      console.log(data);
```
Comment (JVal-NC) [inline src/Api.js]: should this be return rather than console log?

File: src/PostComments.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { postComment } from "./Api";
+
+const PostComments = ({ reviewId, setNewComment }) => {
+  const [comment, setComment] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
```
Comment (JVal-NC) [inline src/PostComments.jsx:19]: Few bugs associated with this component, if an error occurs we get a screen showing the axios error.

1 key example is if we try to post an empty comment, we just an error and "Posting Comment" showing after the error. I think you need a catch block to deal with errors is 1 thing.

You also want to have some way of preventing this kind of request happening (400 errors).

**Review by JVal-NC — COMMENTED** [review]
Hi Fame, really good start. See my comment for something to consider.

**Review by pippaamy — COMMENTED** [review]
That is working much better!

**Comment by JVal-NC** [comment]
Only thing I would say is the message of "Please check your internet connection" when trying to post an empty comment is maybe a bit misleading? Could we have something else?

Noticed this is based on a 400 error from your backend. We generally try to avoid getting 400 errors by validating the inputs on the frontend. Please include something for that and then happy for you to merge.

**Pull request merged**